packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -610,6 +610,7 @@ class _LineItemState extends State<LineItem> {
       if (word != '') {
         try {
           final response = await _debuggerController.evalAtCurrentFrame(word);
+          if (response.json['kind'] == 'error') return;
```

Comment:
I would have thought you could use the type of the response.

Reply:
Yeah that's what I thought too but the type is `Response`.

Reply:
Found a better alternative.